microk8s-resources/default-args/etcd

```diff
@@ -1,3 +1,3 @@
 --data-dir=${SNAP_COMMON}/var/run/etcd
---advertise-client-urls=unix://etcd.socket:2379
---listen-client-urls=unix://etcd.socket:2379
+--advertise-client-urls=http://localhost:2379
+--listen-client-urls=http://localhost:2379
```

Comment:
This is needed due to limitations in the go libraries that etcd uses surrounding UNIX sockets. This is not secure, but AIUI microk8s is being updated to use cert auth.